docs/servers.rst

```diff
@@ -189,10 +189,10 @@ firewall. If you did not, adjust these settings accordingly.
 Continue the installation
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-You can choose whatever username and password you would like. To make
-things easier later you should use the same username and same password
-on both servers (but not the same password as username). Make sure to
-save this password in your admin KeePassX database afterwards.
+You can choose whatever username and passphrase you would like. To make
+things easier later you should use the same username and same passphrase
+on both servers (but not the same passphrase as username). Make sure to
```

Comment:
this is awkward considering now. I think most *nix users set usernames but then set passwords. I dont know of any cases where a *nix user has set a passphrase as a username for the servers. Can we reword this?

Reply:
I'll merge this as is because it is consistent. I agree with you that someone thinking "passphrase" is very unlikely to use the username as a "passphrase". But this is a useful warning, IMHO.